### DISCUSSION TITLE
BirdNET-Pi on Armbian

### DISCUSSION THREAD

**Discussion by jammbrose** [mcguirepr89/BirdNET-Pi#840] [Q&A] [answered]
Has anyone tried running this install script on an Armbian distro?  I tried installing it in Armbian on a Jetson Nano.  The script seems to install without error, and the application seems to run without any apparent error except every bird that is identified doesn't exist in my region.  Is this a Tensorflow issue?

**Comment by matthew73210** [comment]
Hey,
Works fine for me, been running for over 8 months as such. Which model are you running?

**Reply by jammbrose** [reply]
I tried the 6k model as well as the 3k model with no difference in results.

**Reply by jammbrose** [reply]
I'm sorry I should have been more clear.   I was afk.  I am using model BirdNET_GLOBAL_3K_V2.2_Model_FP16.

**Reply by jammbrose** [reply]
The same installation script on Raspberry 64 works perfectly.  Consistent positive IDs.  As I mentioned earlier, the same install script on Armbian Bullseye appears to install correctly and does get positive IDs, just that the IDs are of birds not in my region.  I double-checked the lat lon.  It feels like the app thinks I'm in a different geographic location.

**Reply by matthew73210** [reply, marked as answer]
Looks like what happened to me : https://github.com/mcguirepr89/BirdNET-Pi/discussions/699#discussioncomment-4868157

**Reply by jammbrose** [reply]
This is exactly what was wrong.  I manually updated the labels.txt and everything works.  Why would that break on Armbian?  I can see the labels.txt doesn't change size when the models are toggled.
